gub/3/new.py

```diff
@@ -1,4 +1,4 @@
-# https://docs.python.org/3.0/whatsnew/3.0.html
+# http://docs.python.org/3.0/whatsnew/3.0.html
```

Comment:
reverted it as it was not a URL used as an access-path by the scripts, which was what this change was about